src/pages/revise.js

```diff
@@ -1,49 +1,58 @@
 import React from "react";
-import Grid from '@material-ui/core/Grid';
-import FilterBox from '../components/FilterBox';
-import Flashcard from '../components/Flashcard';
+import Grid from "@material-ui/core/Grid";
+import FilterBox from "../components/FilterBox";
+import Flashcard from "../components/Flashcard";
 import HotkeyBox from "../components/HotkeyBox";
-import {makeStyles} from "@material-ui/core/styles";
+import { makeStyles } from "@material-ui/core/styles";
 import Box from "@material-ui/core/Box";
 
 const useStyles = makeStyles((theme) => ({
-    root: {
-        minHeight: '100vh' - theme.spacing(5),
-        padding: '102px 1em 0em',
-		maxWidth: '2560px',
-		margin: '0 auto',
-    }
+  root: {
+    minHeight: "100vh",
+    padding: "102px 1em 0em",
```

Comment:
Can set this top padding to the set app bar height (or whatever you think is good)
Can get rid of this max width..? Forgot why it was put there

Reply:
Update the padding to matching appbar height